examples/typescript-reactflux/src/actions/TodoActions.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2015, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * TodoActions
+ *
+ * Typescript port by Bernd Paradies, May 2015
+ * @see https://github.com/facebook/flux/blob/master/examples/flux-todomvc/js/actions/TodoActions.js
+ */
```

Comment:
Or at least let them know we did it. Then they don't have to maintain their copy :)